### ISSUE TITLE
[Feature Suggestion] Add  /r/all to side bar

### ISSUE THREAD

**Issue by Mal-content** [joelacus/RedditEnhancer#6]
First of all thanks for extension its amazing and making using the new layout pretty nice. 
I have suggestion to add the link to /r/all to the sidebar did simple mock up:
![image](https://user-images.githubusercontent.com/8986878/225493234-4585db32-3aaa-4dd0-b33d-6cd1e314aaa5.png)

**Comment by joelacus** [comment]
I'm glad you like it :) I'll add this as an option for the next update.

**Comment by joelacus** [comment]
Hi, I noticed at the bottom of the sidebar there is already a link to r/All. 
![image_1](https://user-images.githubusercontent.com/43767965/226148382-ac0e0e52-888b-463f-aa12-7e3ffde0ec5e.png)
I thought instead of two of the same button in the sidebar, a button in the header could be a good idea, as there are already buttons for 'Popular' and 'Happening Now', but not 'All'. I made a concept here, what do you think?
![image_2](https://user-images.githubusercontent.com/43767965/226148416-b17a590b-4690-4d42-8e09-0608f7d03107.png)

**Comment by Mal-content** [comment]
Hey. Having it at the bottom is really inconvenient for quick navigation. But i dont mind having another option in header as well haha. Maybe both to choose from?  Or maybe if you can move header buttons to the left side of the search bar would be amazing (having all navigation grouped in same area is best QoL improvment with those). Made mock up 
![image](https://user-images.githubusercontent.com/8986878/226176315-fad8e62a-0478-491b-bcf5-fbb988ad1f81.png)

**Comment by joelacus** [comment]
I'd not thought about moving them to the left, but yeah I could add that as an option.

**Comment by joelacus** [comment]
Hi. I've just uploaded a new version (0.9.0), it's available now for Firefox, but Chrome and Edge will take about a week for them to verify it. I've added the option to add an 'r/All' button to the header, as well as moving the Feeds section at the bottom of the side menu to the top. Moving the feed header buttons to the left side was giving me some weird bugs, so I haven't include that in this version. Let me know if you encounter any issues. Thanks :)
